xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Note.xml

```diff
@@ -58,343 +58,4 @@ Result :
-      <id>
-        <disabled>0</disabled>
-        <name>id</name>
-        <number>1</number>
-        <prettyName>Macro id</prettyName>
-        <size>30</size>
-        <unmodifiable>0</unmodifiable>
-        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
-      </id>
-      <name>
-        <disabled>0</disabled>
-        <name>name</name>
-        <number>2</number>
-        <prettyName>Macro name</prettyName>
-        <size>30</size>
-        <unmodifiable>0</unmodifiable>
-        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
-      </name>
-      <priority>
-        <disabled>0</disabled>
-        <name>priority</name>
-        <number>11</number>
-        <numberType>integer</numberType>
-        <prettyName>Priority</prettyName>
-        <size>10</size>
-        <unmodifiable>0</unmodifiable>
-        <classType>com.xpn.xwiki.objects.classes.NumberClass</classType>
-      </priority>
-      <supportsInlineMode>
-        <disabled>0</disabled>
-        <displayFormType>select</displayFormType>
-        <displayType>yesno</displayType>
-        <name>supportsInlineMode</name>
-        <number>5</number>
-        <prettyName>Supports inline mode</prettyName>
-        <unmodifiable>0</unmodifiable>
-        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
-      </supportsInlineMode>
-      <visibility>
-        <cache>0</cache>
-        <disabled>0</disabled>
-        <displayType>select</displayType>
-        <freeText>forbidden</freeText>
-        <largeStorage>0</largeStorage>
-        <multiSelect>0</multiSelect>
-        <name>visibility</name>
-        <number>6</number>
-        <prettyName>Macro visibility</prettyName>
-        <relationalStorage>0</relationalStorage>
-        <separator>|</separator>
-        <separators>|</separators>
-        <size>1</size>
-        <unmodifiable>0</unmodifiable>
-        <values>Current User|Current Wiki|Global</values>
-        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
-      </visibility>
-    </class>
-    <property>
-      <async_cached>0</async_cached>
-    </property>
-    <property>
-      <async_context/>
-    </property>
-    <property>
-      <async_enabled>0</async_enabled>
-    </property>
-    <property>
-      <code>{{velocity output="false"}}
-#macro (executeMacro)
-  {{warning}}
-    #if("$!wikimacro.parameters.title" != "")
-    **{{wikimacroparameter name="title" /}}**
-
-    #end
-    {{wikimacrocontent /}}
-  {{/warning}}
-#end
-{{/velocity}}
-
-{{include reference="Licenses.Code.VelocityMacros"/}}
-
-{{velocity}}
-## We need to check if there is a valid license because the macro is registered even if the user doesn't have view right
-## on the macro definition page. See XWIKI-14828: Rendering macros defined in wiki pages are available to users that
-## don't have view right on those pages.
-#if ($services.licensing.licensor.hasLicensureForEntity($xcontext.macro.doc.documentReference))
```

Comment:
This check seems to be lost in your implementation. This means you will be able to use the macro after the license expires.